lang/en.json

```diff
@@ -357,12 +357,14 @@
         },
         "payment_methods": {
             "heading": "Payment Methods",
+            "payment_method": "Payment method",
+            "billing_address": "Billing address",
             "card_ending_in": "ending in {last_four}",
             "card_expiry": "{month}/{year}",
             "new_payment_method": "Add new payment method",
             "no_methods": "You currently have no payment methods added to your account",
-            "name_on_card": "Name on card",
```

Comment:
So we no longer be showing name on the card ?

Reply:
Hey @junedkazi , yes we realized the ```name on card``` was not something we keep when vaulting the card. Hey @jackosaurus , please correct me if I'm wrong.

Reply:
That's correct, we do not store "Name on card" (we were mistaken)